x-pack/plugins/ml/server/models/data_recognizer/modules/network/ml/high_count_network_events.json

```diff
@@ -0,0 +1,34 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "A machine learning job detected an unusually large spike in network traffic. Such a burst of traffic, if not caused by a surge in business activity, can be due to suspicious or malicious activity. Large-scale data exfiltration may produce a burst of network traffic; this could also be due to unusually large amounts of reconnaissance or enumeration traffic.  Denial-of-service attacks or traffic floods may also produce such a surge in traffic.",
```

Comment:
As above, for consistency of style with the other security jobs, I'd suggest changing the first part of the description here to 

```
Security: Network - Looks for unusually large spikes in network traffic.
```